examples/react-native-typescript/tsconfig.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
please add here

```
"lerna": {
    "disabled": true
  },
```

Otherwise, this project will participate in the ts distribution